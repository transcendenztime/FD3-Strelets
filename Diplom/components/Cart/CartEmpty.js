import React from 'react';

class CartEmpty extends React.PureComponent {
          
  render() {

    return (
        <div className = "cart-empty">
          <h3 className="page-subtitle">
            В вашей корзине пусто!
          </h3>
          <p>
           Для добавления продуктов в корзину нажмите на кнопку с изображением корзины "<i className="fas fa-shopping-cart"></i> 
            " рядом с выбранным продуктом           
          </p>
        </div>
        
    );
    
  }

}
    
export default CartEmpty;