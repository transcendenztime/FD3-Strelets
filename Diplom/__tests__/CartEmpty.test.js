import React from 'react';
import renderer from 'react-test-renderer';

import CartEmpty from '../components/Cart/CartEmpty';

test('внешний вид CartEmpty', () => {

    // создаём тестовую версию компонента
    const component = renderer.create(
          <CartEmpty />
    );
  
    // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
    let componentTree=component.toJSON();
    expect(componentTree).toMatchSnapshot();
          
  });