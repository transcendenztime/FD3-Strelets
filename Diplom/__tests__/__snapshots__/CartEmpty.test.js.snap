// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`внешний вид CartEmpty 1`] = `
<div
  className="cart-empty"
>
  <h3
    className="page-subtitle"
  >
    В вашей корзине пусто!
  </h3>
  <p>
    Для добавления продуктов в корзину нажмите на кнопку с изображением корзины "
    <i
      className="fas fa-shopping-cart"
    />
    " рядом с выбранным продуктом
  </p>
</div>
`;
